### PR TITLE
Fix to deprecated function call

### PR DESCRIPTION
**Pull request by aSeriousDeveloper**
I received warnings in my logs about a deprecated function call:

```
LOG.warning: rtrim(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/vendor/ueberdosis/tiptap-php/src/Utils/HTML.php on line 28
```

On this same line, I noticed there's an already existing coalesce for an `rtrim` function, for the style attribute.
I simply did the same method of coalescing the `value` variable to an empty string if it's `null`.

### CODE REVIEW

**Comment by aSeriousDeveloper** [comment]
Hi, any chance this can get reviewed? I'm receiving a fair amount of deprecation warnings in logs for my projects and this fix would help resolve them.

**Review by silasjoisten — APPROVED** [review]

File: src/Utils/HTML.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public static function mergeAttributes()
 
                 // style="color: red;"
                 if ($key === 'style') {
-                    $style = rtrim($attributes['style'] ?? '', '; ') . '; ' . rtrim($value, ';') . '; ';
+                    $style = rtrim($attributes['style'] ?? '', '; ') . '; ' . rtrim($value ?? '', ';') . '; ';
```
Comment (timoisik) [inline src/Utils/HTML.php:28]: Thank you very much for your contribution.

If I put in something like:
```php
$attributes = [
    ['class' => 'a', 'style' => 'color: red;'],
    ['class' => 'b', 'style' => null],
];

$result = HTML::mergeAttributes(...$attributes);
```
I do get something like this:
```php
[
  'class' => 'a b',
  'style' => 'color: red; ;',
]
```

While this was there before too and would probably might not cause significant issues, I wonder if it make sense to just add something like 
```php
if (!$value) { continue; }
```
in the beginning of the second loop.

What do you think? 

Could you also add some more test cases so we can make sure that everything works as expected also with `null` values in there?

Comment (aSeriousDeveloper) [reply src/Utils/HTML.php:28]: The fix proposed here is more to prevent the deprecation warning & eventual break come PHP 9 as mentioned in this RFC: https://wiki.php.net/rfc/deprecate_null_to_scalar_internal_arg

Currently, the `rtrim` function converts `null` into an empty string, which from PHP 8.1 provides a deprecation warning. This PR instead performs this conversion before passing calling `rtrim` via null coalescence, providing what should be the same functionality but without a deprecation warning or a potential thrown exception come PHP 9.

I could expand the scope of the PR to better refactor this function and add the test cases to support this, but that might be worth a separate PR in the future to clean up this part of the code instead.

Comment (timoisik) [reply src/Utils/HTML.php:28]: Yes, thought skipping if the value is null also prevents the warning. However, this PR solves one issue. Thanks for your contribution. Would love another PR with adding tests.
